### PR TITLE
compatibility with JQuery3

### PR DESCRIPTION
**Pull request by live627**
Replaced deprecated $.(un)load, $.(un)bind with $.on and $.off

### CODE REVIEW

**Comment by live627** [comment]
DuckDuckGo has failed again :stuck_out_tongue_winking_eye:

**Comment by Spuds** [comment]
I have hated <![CDATA[ // ]] for so very long ... thank you thank you !!

**Comment by live627** [comment]
Yeah, those were invented by cave men using their Commodore 64.

File: themes/default/scripts/admin.js [hunk 1]
```diff
@@ -995,7 +995,7 @@ function select_in_category(operation, brd_list)
  * Server Settings > Caching, toggles input fields on/off as appropriate for
  * a given cache engine selection
  */
-$(document).ready(function() {
+$(function() {
 	$('#cache_accelerator').change(function() {
```
Comment (Spuds) [inline themes/default/scripts/admin.js]: Is this now recommended for 3.0? 

I know $(function() is the shorthand for $(document).ready, although the later has better readability IMO ... that said  I'm fine with either and less typing is always better.  Just curious if this was another recommended 3.0 change that I missed 😇 they are awesome at hiding little depreciated things in the docs.

**Comment by live627** [comment]
> Is this now recommended for 3.0?

 Yes https://api.jquery.com/ready/

**Comment by emanuele45** [comment]
> Yeah, those were invented by cave men using their Commodore 64.

but... but... they were a piece of history!! :sob: 
Who cares, history is written by the last one standing. :stuck_out_tongue:

**Pull request merged**